---
layout: page
title: "RESEARCH"
description: "Lu Lab"
active: about
header-img: "img/research.jpg"
---
<style>
.small{
    line-height: 200%;
    font-size: 24px;
}
.big{
    line-height: 65px;
}
.line{
    height:2px;border:none;border-top:1px solid #000000;
}
</style>
<body>

<h1>What we do</h1>
<hr class="line">
<p class="small">To speed up breeding cycle and further improve crop varieties, we are trying to understand the genetics of agronomic traits, especially the complex traits (e.g. yield, heterosis, resistance, etc). 
    We use quantitative genetics and statistical genomics approaches to answer the following questions.
</p>

<p class="big">
1) What loci are controlling the traits?
<br>
2) What is the effect distribution among these loci?
<br>
3) What is the casual variant in each locus?
<br>
4) What variants interact with environments differently?
<br>
5) How to apply the knowledge obtained from 1-4 efficiently to accelerate breeding?
<br></p>
<p class="small">Our research goal is to improve genomic selection and/or develop a new system for breeding by using genomic editing technologies efficiently.
</p>
<br>
<h1>Challenges</h1>
<hr class="line">
<p class="small">Quantitative traits are important for crop improvement, but difficult to work with, because 1) hundreds to thousands of loci are controlling the traits; 2) each locus has small effect. 
    The challenge is to identify thousands of loci and accurately estimate their effects together, which generally requires a large sample size in GWAS. However, due to the relatively higher cost for collecting phenotypes (compared with human genetic studies), it is not feasible to build a population of 1 million plants to estimate these small effects. 
    There has to be another route to dissect complex trait, which needs to be creative, efficient, and cost-effective.</p>
<img src="challenge.jpg">
<br>
<h1>Our Solution</h1>
<hr class="line">
<p class="small">We use functional genome prediction approach to pinpoint trait controlling variants and estimate their effect. 
    By using evolution (natureâ€™s experiment one billion years), decomposing complex traits to molecular level, combining with field trials and high-throughput sequencing, utilizing the power of optimization and machine learning approaches, we predict causal variants underneath important agronomic traits. 
    The successful development of functional genome prediction approach will tremendously expand the application of genomic editing technologies in crops.</p>








<br>
<br>
<br>
<br>
<br>


</body>


