---
layout: page
title: "Journal Club"
description: "Plant Genetics Lab"
active: about
header-img: "rt/silhouette-g53e47a572_1280.jpeg"
---


<style>
    .img{
        width: 25%;
        height: 25%;
        cursor: pointer;
    }
    .line{
        height:2px;border:none;border-top:1px solid #000000;
    }
    .line2{
        height:2px;border:none;border-top:1px solid darkgray; 
    }
    a,a:hover{
        color:#5176aa;
    }
    a,a:focus{
        color: darkorange;
    }
    </style>

    <body>
<h1>Journal Club</h1>
<hr class="line">
<br>
<h2>2022</h2>
<p>
    <h3>Mar 25,2022</h3>
    Presenter: Jun Xu
    Paper: Vaishnav, E.D., de Boer, C.G., Molinet, J. et al. The evolution, evolvability and engineering of gene regulatory DNA. Nature 603, 455–463 (2022). 
    <br><a href="https://doi.org/10.1038/s41586-022-04506-6">https://doi.org/10.1038/s41586-022-04506-6</a>
</p>
<p>
    <h3>Mar 18,2022</h3>
    Presenter: Lipeng Kang
    Paper: Powell, D.L.; García-Olazábal, M.; Keegan, M.; Reilly, P.; Du, K.; Díaz-Loyo, A.P.; Banerjee, S.; Blakkan, D.; Reich, D.; Andolfatto, P.; et al. Natural Hybridization Reveals Incompatible Alleles That Cause Melanoma in Swordtail Fish. Science (80-. ). 2020, 368, 731–736.
    <a href="https://www.science.org/doi/abs/10.1126/science.aba5216">https://www.science.org/doi/abs/10.1126/science.aba5216</a>
</p>
<p>
    <h3>Mar 04,2022</h3>
    Presenter: Beirui Niu<br>
    Paper:He, F., Wang, W., Rutter, W.B. et al. Genomic variants affecting homoeologous gene expression dosage contribute to agronomic trait variation in allopolyploid wheat. Nat Commun 13, 826 (2022). 
    <br><a href="https://doi.org/10.1038/s41467-022-28453-y">https://doi.org/10.1038/s41467-022-28453-y</a>
</p>
<p>
    <h3>Feb 25,2022</h3>
    Presenter: Zelin Niu<br>
    Paper: Monroe, J.G., Srikant, T., Carbonell-Bejerano, P. et al. Mutation bias reflects natural selection in Arabidopsis thaliana. Nature 602, 101–105 (2022). 
    <br><a href="https://doi.org/10.1038/s41586-021-04269-6">https://doi.org/10.1038/s41586-021-04269-6</a>
</p>
<p>
    <h3>Jan 14,2022</h3>
    Presenter: Xuebing Qiu<br>
    Paper: Laricchia, K. M. et al. Mitochondrial DNA variation across 56,434 individuals in gnomAD. bioRxiv 2021.07.23.453510 (2021)
    <br><a href="https://doi.org/10.1101/2021.07.23.453510">https://doi.org/10.1101/2021.07.23.453510</a>
</p>
<hr class="line">
<br>
<h2>2021</h2>
<p>
    <h3>Dec 31,2021</h3>
    Presenter: Hangyuan Cheng<br>
    Paper: Zheng, J., Guo, N. & Wagner, A. Selection enhances protein evolvability by increasing mutational robustness and foldability. Science (80-. ). 370, eabb5962 (2020).
    <br><a href="https://www.science.org/doi/10.1126/science.abb5962">https://www.science.org/doi/10.1126/science.abb5962</a>
</p>
<p>
    <h3>Dec 24,2021</h3>
    Presenter: Hua Zhang<br>
    Paper: Yonova-Doing, E., Calabrese, C., Gomez-Duran, A. et al. An atlas of mitochondrial DNA genotype–phenotype associations in the UK Biobank. Nat Genet 53, 982–993 (2021). 
    <br><a href="https://doi.org/10.1038/s41588-021-00868-1">https://doi.org/10.1038/s41588-021-00868-1</a>
</p>
<p>
    <h3>Dec 10,2021</h3>
    Presenter: Xuebo Zhao<br>
    Paper: Bergström, A., Stringer, C., Hajdinjak, M. et al. Origins of modern human ancestry. Nature 590, 229–237 (2021). 
    <br><a href="https://doi.org/10.1038/s41586-021-03244-5">https://doi.org/10.1038/s41586-021-03244-5</a>
</p>
<p>
    <h3>Dec 03,2021</h3>
    Presenter: Jiayu Dong<br>
    Paper: Benjamin M Peter, Admixture, Population Structure, and F-Statistics, Genetics, Volume 202, Issue 4, 1 April 2016, Pages 1485–1501.
    <br><a href="https://doi.org/10.1534/genetics.115.183913">https://doi.org/10.1534/genetics.115.183913</a>
</p>
<p>
    <h3>Nov 26,2021</h3>
    Presenter: Liping Jiang<br>
    Paper: Avsec, Ž., Agarwal, V., Visentin, D. et al. Effective gene expression prediction from sequence by integrating long-range interactions. Nat Methods 18, 1196–1203 (2021). 
    <br><a href="https://doi.org/10.1038/s41592-021-01252-x">https://doi.org/10.1038/s41592-021-01252-x</a>
</p>
<p>
    <h3>Nov 19,2021</h3>
    Presenter: Xiaohan Yang<br>
    Paper: Bonder, M.J., Smail, C., Gloudemans, M.J. et al. Identification of rare and common regulatory variants in pluripotent cells using population-scale transcriptomics. Nat Genet 53, 313–321 (2021). 
    <br><a href="https://doi.org/10.1038/s41588-021-00800-7">https://doi.org/10.1038/s41588-021-00800-7</a>
</p>
<p>
    <h3>Nov 12,2021</h3>
    Presenter: Song Xu<br>
    Paper: Wong, A.K., Sealfon, R.S.G., Theesfeld, C.L. et al. Decoding disease: from genomes to networks to phenotypes. Nat Rev Genet 22, 774–790 (2021). 
    <br><a href="https://doi.org/10.1038/s41576-021-00389-x">https://doi.org/10.1038/s41576-021-00389-x</a>
</p>
<p>
    <h3>Nov 05,2021</h3>
    Presenter: Daxing Xu<br>
    Paper: Frazer, J., Notin, P., Dias, M. et al. Disease variant prediction with deep generative models of evolutionary data. Nature 599, 91–95 (2021). 
    <br><a href="https://doi.org/10.1038/s41586-021-04043-8">https://doi.org/10.1038/s41586-021-04043-8</a>
</p>
<p>
    <h3>Oct 28,2021</h3>
    Presenter: Aoyue Bi<br>
    Paper: Li, JT., Wang, Q., Huang Yang, MD. et al. Parallel subgenome structure and divergent expression evolution of allo-tetraploid common carp and goldfish. Nat Genet 53, 1493–1503 (2021). 
    <br><a href="https://doi.org/10.1038/s41588-021-00933-9">https://doi.org/10.1038/s41588-021-00933-9</a>
</p>
<p>
    <h3>Sep 30,2021</h3>
    Presenter: Ming Zhang<br>
    Paper: Harvey, M. G. et al. The evolution of a tropical biodiversity hotspot. Science (80-. ). 370, 1343–1348 (2020).
    <br><a href="https://www.science.org/doi/10.1126/science.aaz6970">https://www.science.org/doi/10.1126/science.aaz6970</a>
</p>
<p>
    <h3>Sep 24,2021</h3>
    Presenter: Yafei Guo<br>
    Paper: Fuller, Z. L. et al. Population genetics of the coral Acropora millepora: Toward genomic prediction of bleaching. Science (80-. ). 369, eaba4674 (2020).
    <br><a href="https://www.science.org/doi/10.1126/science.aba4674">https://www.science.org/doi/10.1126/science.aba4674</a>
</p>
<p>
    <h3>Sep 17,2021</h3>
    Presenter: Beirui Niu<br>
    Paper: Stitzer, M. C., Anderson, S. N., Springer, N. M. & Ross-Ibarra, J. The genomic ecosystem of transposable elements in maize. PLOS Genet. 17, e1009768 (2021).
    <br><a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009768">https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009768</a>
</p>
<p>
    <h3>Sep 03,2021</h3>
    Presenter: Xinyue Song<br>
    Paper: Kuzmin, Elena et al. “Exploring whole-genome duplicate gene retention with complex genetic interaction analysis.” Science (New York, N.Y.) vol. 368,6498 (2020): eaaz5667. 
    <br><a href="DOI: 10.1126/science.aaz5667">DOI: 10.1126/science.aaz5667</a>
</p>
<br>

<br>
<br>
<br>

    </body>