---
layout: page
title: "PUBLICATION"
description: "Plant Genetics Lab"
active: about
header-img: "img/paper.jpg"
---

<style>
.xie{
    font-weight: bold;
    font-style: oblique;   
}
.xie2{
    font-style: oblique;
}
.bold{
    font-weight: bold;
}
.subhead{
    color: darkgrey;
}
a,a:hover{
    color:skyblue;
}
a,a:focus{
    color: orange;
}

</style>
<h2><a href="https://scholar.google.com/citations?user=RbNQWbUAAAAJ&hl=en"><bold>Google Scholar</bold></a></h2>
<br>

<h2>SELECTED PUBLICATIONS</h2>
<h3>(*Corresponding author; #Co-first author)</h3>

<p>
    6. Xuebo Zhao#, Yafei Guo#, Lipeng Kang, Changbin Yin, Aoyue Bi, Daxing Xu, Zhiliang Zhang, Zhang Jijin, 
    Xiaohan Yang, Jun Xu, Song Xu, Xinyue Song, Ming Zhang, Yiwen Li, Philip Kear, Jing Wang, Zhiyong Liu, Xiangdong Fu, 
    <span class="bold">Fei Lu</span>. Population genomics unravels the Holocene history of bread wheat and its relatives.
    <span class="xie">Nature Plants. </span>(2023). 
    <br><a href=" https://doi.org/10.1038/s41477-023-01367-3">doi:10.1038/s41477-023-01367-3</a>

</p>

<p>
    5. Yuanying Peng#,*, Honghai Yan#, Laichun Guo#, Cao Deng#, Chunlong Wang#, Yubo Wang#, Lipeng Kang, Pingping Zhou, Kaiquan Yu, Xiaolong Dong, Xiaomeng Liu, Zongyi Sun, 
    Yun Peng, Jun Zhao, Di Deng, Yinghong Xu, Ying Li, Qiantao Jiang, Yan Li, Liming Wei, Jirui Wang, Jian Ma, Ming Hao, Wei Li, Houyang Kang, Zhengsong Peng, Dengcai Liu, 
    Jizeng Jia, Youliang Zheng, Tao Ma*, Yuming Wei*, <span class="bold">Fei Lu*</span>, Changzhong Ren*. 
    Reference genome assemblies reveal the origin and evolution of allohexaploid oat.
    <span class="xie">Nature Genetics. </span>(2022). 
    <br><a href="https://www.nature.com/articles/s41588-022-01127-7">doi:10.1038/s41588-022-01127-7.</a>
</p>

<p>
    4. Yao Zhou#, Xuebo Zhao#, Yiwen Li, Jun Xu, Aoyue Bi, Lipeng Kang, Daxing Xu, Haofeng Chen, Ying Wang, Yuan-ge Wang, Sanyang Liu, Chengzhi Jiao, Hongfeng Lu, Jing Wang, Changbin Yin, Yuling Jiao*, <span class="bold">F. Lu*</span>.
    <span class="xie2">Triticum</span> population sequencing provides insights into wheat adaptation. 
    <span class="xie">Nature Genetics. </span>(2020). 
    <br><a href="https://www.nature.com/articles/s41588-020-00722-w">doi:10.1038/s41588-020-00722-w.</a>
</p>

<p>
    3. Punna Ramu*, Williams Esuma, Robert Kawuki, Ismail Y Rabbi, Chiedozie Egesi, Jessen V Bredeson, Rebecca S Bart, Janu Verma, Edward S Buckler, <span class="bold">Fei Lu*</span>.
    Cassava haplotype map highlights fixation of deleterious mutations during clonal propagation. 
    <span class="xie">Nature Genetics. </span>(2017).
   <br> <a href="https://www.nature.com/articles/ng.3845">doi:10.1038/ng.3845</a>
</p>

<p>
    2. <span class="bold">Fei Lu*</span>, Maria C Romay, Jeff C Glaubitz, Peter J Bradbury, Rob J Elshire, Tianyu Wang, Yu Li, Yongxiang Li, Kassa Semagn, Xuecai Zhang, Alvaro G. Hernandez, Mark A. Mikel, Ilya Soifer, Omer Barad, Edward S Buckler*. 
    High-resolution genetic mapping of maize pan-genome sequence anchors. 
    <span class="xie">Nature Communications. </span>(2015).
    <br><a href="https://www.nature.com/articles/ncomms7914 ">doi: 10.1038/ncomms7914. </a>
</p>

<p>
   1.<span class="bold"> Lu F</span>, Lipka AE, Elshire RJ, Glaubitz JC, Cherney JH, Casler MD, Buckler ES, Costich DE.
    Switchgrass genomic diversity, ploidy and evolution: novel insights from a network-based SNP discovery protocol. 
    <span class="xie"> PLoS Genetics. </span>(2013).
    <br><a href="https://doi.org/10.1371/journal.pgen.1003215">https://doi.org/10.1371/journal.pgen.1003215</a>
    (Citation: <span class="bold">675</span> by 2022-07-14)
</p>

<br>

<h2>PREPRINTS</h2>
<h3>(*Corresponding author; #Co-first author)</h3>
<p>
    9. Zhiliang Zhang#, Jijin Zhang#, Lipeng Kang, Xuebing Qiu, Song Xu, Jun Xu, Yafei Guo, Zelin Niu, Beirui Niu, Aoyue Bi, Xuebo Zhao, Daxing Xu, Jing Wang, Changbin Yin, 
    <span class="bold">Fei Lu</span>.Structural variation discovery in wheat using PacBio high-fidelity sequencing.
    <span class="xie">bioRxiv. </span>(2023).
    <br><a href="https://www.biorxiv.org/content/10.1101/2023.12.08.570887v1"> doi: 10.1101/2023.12.08.570887</a>
</p> 


<p>
    8. Aoyue Bi#, Daxing Xu#, Lipeng Kang, Yafei Guo, Xinyue Song, Xuebo Zhao, Jijin Zhang, Zhiliang Zhang, Yiwen Li, Changbin Yin, Jing Wang, 
    <span class="bold">Fei Lu</span>. An integrated map of genetic variation from 1,062 wheat genomes. 
    <span class="xie">bioRxiv. </span>(2023).
    <br><a href="https://www.biorxiv.org/content/10.1101/2023.03.31.535022v1"> doi:10.1101/2023.03.31.535022</a>
</p>    

<p>
    7. Xuebo Zhao#, Yafei Guo#, Lipeng Kang, Aoyue Bi, Daxing Xu, Zhiliang Zhang, Zhang Jijin, Xiaohan Yang, Jun Xu, Song Xu, Xinyue Song, Ming Zhang, Yiwen Li, Philip Kear, Jing Wang, Changbin Yin, Zhiyong Liu, Xiangdong Fu, <span class="bold">Fei Lu*</span>.
    Population genomics unravels the Holocene history of <span class="xie2">Triticum-Aegilops</span> species.
    <span class="xie">bioRxiv. </span>(2022).
    <br><a href="https://www.biorxiv.org/content/10.1101/2022.04.07.487499v1"> doi:10.1101/2022.04.07.487499</a>
</p>

<p>
    6. Zhiliang Zhang#, Jijin Zhang#, Lipeng Kang, Xuebing Qiu, Beirui Niu, Aoyue Bi, Xuebo Zhao, Daxing Xu, Jing Wang, Changbin Yin, Xiangdong Fu, <span class="bold">Fei Lu*</span>.
    Genotyping of structural variation using PacBio high-fidelity sequencing.
    <span class="xie">bioRxiv. </span>(2021).
    <br><a href="https://www.biorxiv.org/content/10.1101/2021.10.28.466362v1.full">doi:10.1101/2021.10.28.466362</a>
</p>

<p>
    5. Yuanying Peng#, Honghai Yan#, Laichun Guo, Cao Deng, Lipeng Kang, Chunlong Wang, Pingping Zhou, Kaiquan Yu, Xiaolong Dong, Jun Zhao, Yun Peng, Xiaomeng Liu, Di Deng, 
    Yinghong Xu, Ying Li, Qiantao Jiang, Yan Li, Liming Wei, Jirui Wang, Jian Ma, Ming Hao, Wei Li, Houyang Kang, Youliang Zheng, Tao Ma*, Yuming Wei*, <span class="bold">Fei Lu*</span>, Changzhong Ren*. 
    Reference genome assemblies reveal the origin and evolution of allohexaploid oat. 
    <span class="xie">Research Square. </span> (2021).
    <br><a href="https://europepmc.org/article/ppr/ppr363254"> doi:10.21203/rs.3.rs-664692/v1</a>
</p>

<p>
    4. Xuebo Zhao, Xiangdong Fu, Changbin Yin*, <span class="bold">Fei Lu*</span>.Wheat Speciation and Adaptation: Perspectives from Reticulate Evolution. 
    <span class="xie">Preprints. </span>(2021).
    <br><a href="https://doi.org/10.1007/s42994-021-00047-0">https://doi.org/10.1007/s42994-021-00047-0</a>
</p>

<p>
    3. Yao Zhou, Xuebo Zhao, Yiwen Li, Jun Xu, Aoyue Bi, Lipeng Kang, Haofeng Chen, Ying Wang, Yuan-ge Wang, Sanyang Liu, Chengzhi Jiao, Hongfeng Lu, Jing Wang, Changbin Yin, Yuling Jiao*, < span class="bold">Fei Lu*</span>,
    Convergence within divergence: insights of wheat adaptation from <span class="xie2">Triticum </span> sequencing.
    <span class="xie">bioRxiv. </span>(2020).
    <br><a href="https://www.biorxiv.org/content/10.1101/2020.03.21.001362v1">doi:10.1101/2020.03.21.001362</a>
</p>

<p>
    2. Haofeng Chen, Chengzhi Jiao, Ying Wang, Yuange Wang, Caihuan Tian, Haopeng Yu, Jing Wang, Xiangfeng Wang, <span class="bold">Fei Lu</span>, Xiangdong Fu, Yongbiao Xue, Wenkai Jiang, Hongqing Ling, Hongfeng Lu, Yuling Jiao.
    Comparative Population Genomics of Bread Wheat (<span class="xie2">Triticum aestivum</span>) Reveals Its Cultivation and Breeding History in China.
    <span class="xie">bioRxiv. </span>(2019).
    <br><a href="https://doi.org/10.1101/519587">doi:10.1101/519587</a>
</p>


<p>
    1. Punna Ramu*, Williams Esuma, Robert Kawuki, Ismail Y Rabbi, Chiedozie Egesi, Jessen V Bredeson, Rebecca S Bart, Janu Verma, Edward S Buckler, <span class="bold">Fei Lu*</span>.
    Cassava HapMap: Managing genetic load in a clonal crop species. 
    <span class="xie">bioRxiv. </span>(2016).
    <br><a href="https://www.biorxiv.org/content/10.1101/077123v1">doi:http://dx.doi.org/10.1101/077123</a>
</p>


<br>
<h2>PEER-REVIEWED PUBLICATIONS</h2>
<h3>(*Corresponding author; #Co-first author)</h3>

<p>
    23. Liping Shen#, Lili Zhang#, Changbin Yin#, Xiaowan Xu#, Yangyang Liu, Kuocheng Shen, He Wu, Zhiwen Sun, Ke Wang, Zhonghu He, Xueyong Zhang, Chenyang Hao, Jian Hou, Aoyue Bi, Xuebo Zhao, Daxing Xu, Botao Ye, Xuchang Yu, Ziying Wang, Danni Liu, Yuanfeng Hao, 
    <span class="bold">Fei Lu</span>, Zifeng Guo*. The wheat sucrose synthase gene TaSus1 is a major determinant of grain number per spike. 
    <span class="xie">The Crop Journal. </span>(2023). 
    <br><a href="https://www.sciencedirect.com/science/article/pii/S2214514123001721">doi: 10.1016/j.cj.2023.11.007</a>

</p>

<p>
    22. Yafei Guo,
     <span class="bold">Fei Lu</span>. The changing colour of carrot.
    <span class="xie">Nature Plants. </span>(2023).
    <br><a href="https://www.nature.com/articles/s41477-023-01523-9#:~:text=Increasing%20carotenoid%20content%20has%20led,red%20tones%20of%20improved%20cultivars">doi: 10.1038/s41477-023-01523-9.</a>
</p> 
<p>
    21. Yangyang Liu#, Jun Chen#, Changbin Yin#, Ziying Wang, He Wu, Kuocheng Shen, Zhiliang Zhang, Lipeng Kang, Song Xu, Aoyue Bi, Xuebo Zhao, Daxing Xu, Zhonghu He, Xueyong Zhang, Chenyang Hao, Jianhui Wu, Yan Gong, Xuchang Yu, Zhiwen Sun, Botao Ye, Danni Liu, Lili Zhang, Liping Shen, Yuanfeng Hao*, Youzhi Ma*,
     <span class="bold">Fei Lu*</span>, Zifeng Guo*. A High-Resolution Genotype–Phenotype Map Identifies the TaSPL17 Controlling Grain Number and Size in Wheat. 
    <span class="xie">Genome Biology. </span>(2023).
    <br><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03044-2">doi: 10.1186/s13059-023-02932-x.</a>
</p> 

<p>
    20. Xuebo Zhao, Yafei Guo, <span class="bold">Fei Lu*</span>. Reply to: Where Was Wheat Domesticated?.
    <span class="xie">Nature Plants. </span>(2023).
    <br><a href="https://www.nature.com/articles/s41477-023-01468-z">doi: 10.1038/s41477-023-01468-z.</a>
</p> 
<p>
    19. Haoxu Gao, Jianxin Bian, <span class="bold">Fei Lu</span>, Yuling Jiao*, Hang He*. Triticeae Crop Genome Biology: An Endless Frontier. 
    <span class="xie">Frontiers in Plant Science. </span>(2023).
    <br><a href="https://www.frontiersin.org/articles/10.3389/fpls.2023.1222681/full"> doi: 10.3389/FPLS.2023.1222681/BIBTEX.</a>
</p> 

<p>
    18. Yangyang Liu#, Kuocheng Shen#, Changbin Yin#, Xiaowan Xu#, Xuchang Yu, Botao Ye, Zhiwen Sun, Jiayu Dong, Aoyue Bi, 
    Xuebo Zhao, Daxing Xu, Zhonghu He, Xueyong Zhang, Chenyang Hao, Jianhui Wu6, Ziying Wang, He Wu, Danni Liu, Lili Zhang, 
    Liping Shen, Yuanfeng Hao*, <span class="bold">Fei Lu*</span>, Zifeng Guo*. Genetic basis of geographical differentiation and breeding selection for wheat plant architecture traits. 
    <span class="xie">Genome Biology. </span>(2023).
    <br><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-02932-x">doi:10.1186/s13059-023-02932-x</a>
</p>    
<p>
    17. Xuebo Zhao#, Yafei Guo#, Lipeng Kang, Changbin Yin, Aoyue Bi, Daxing Xu, Zhiliang Zhang, Zhang Jijin, 
    Xiaohan Yang, Jun Xu, Song Xu, Xinyue Song, Ming Zhang, Yiwen Li, Philip Kear, Jing Wang, Zhiyong Liu, Xiangdong Fu, 
    <span class="bold">Fei Lu</span>. Population genomics unravels the Holocene history of bread wheat and its relatives.
    <span class="xie">Nature Plants. </span>(2023). 
    <br><a href=" https://doi.org/10.1038/s41477-023-01367-3">doi:10.1038/s41477-023-01367-3</a>
</p>
<p>
    16. Hao Jiang#, Yu Fang#, Dong Yan, Si-tong Liu, Jun Wei, Fei-long Guo, Xing-ting Wu, Hong Cao, Chang-bin Yin, 
    <span class="bold"> Fei Lu</span>, Li-feng Gao*, Yong-xiu Liu*. (2022).
    Genome-wide association study reveals a NAC transcription factor TaNAC074 linked to pre-harvest sprouting tolerance in wheat. 
    <span class="xie">Theor. Appl. Genet. </span> (2022). 
    <br><a href="https://doi.org/10.1007/s00122-022-04184-y">doi:10.1007/s00122-022-04184-y.</a>
</p>

<p>
    15. Jun Xiao*, Bao Liu, Yingyin Yao, Zifeng Guo, Haiyan Jia, Lingrang Kong, Aimin Zhang, Wujun Ma, Zhongfu Ni, Shengbao Xu, <span class="bold"> Fei Lu</span>, 
    Yuannian Jiao, Wuyun Yang, Xuelei Lin, Silong Sun, Zefu Lu, Lifeng Gao, Guangyao Zhao, Shuanghe Cao, Qian Chen, Kunpu Zhang, Mengcheng Wang, Meng Wang, 
    Zhaorong Hu, Weilong Guo, Guoqiang Li, Xin Ma, Junming Li, Fangpu Han, Xiangdong Fu, Zhengqiang Ma, Daowen Wang*, Xueyong Zhang*, Hong-Qing Ling*, Guangmin Xia*, 
    Yiping Tong*, Zhiyong Liu*, Zhonghu He*, Jizeng Jia*, Kang Chong*.
    Wheat genomic study for genetic improvement of traits in China.
    <span class="xie">Sci. China Life Sci. </span> (2022). 
    <br><a href="https://doi.org/10.1007/s11427-022-2178-7">doi:10.1007/s11427-022-2178-7.</a>
</p>

<p>
    14. Yuanying Peng#,*, Honghai Yan#, Laichun Guo#, Cao Deng#, Chunlong Wang#, Yubo Wang#, Lipeng Kang, Pingping Zhou, Kaiquan Yu, Xiaolong Dong, Xiaomeng Liu, Zongyi Sun, Yun Peng, Jun Zhao, Di Deng, Yinghong Xu, Ying Li, Qiantao Jiang, 
    Yan Li, Liming Wei, Jirui Wang, Jian Ma, Ming Hao, Wei Li, Houyang Kang, Zhengsong Peng, Dengcai Liu, Jizeng Jia, Youliang Zheng, Tao Ma*, Yuming Wei*, <span class="bold">Fei Lu*, </span>Changzhong Ren*.
    Reference genome assemblies reveal the origin and evolution of allohexaploid oat. 
    <span class="xie">Nature Genetics. </span> (2022).
    <br><a href="https://www.nature.com/articles/s41588-022-01127-7">doi:10.1038/s41588-022-01127-7.</a>
</p>

<p>
    13. Yuange Wang#, Fei Du#, Jian Wang, Ke Wang, Caihuan Tian, Xiaoquan Qi,<span class="bold"> Fei Lu</span>, Xigang Liu, Xingguo Ye, and Yuling Jiao*. 
    Improving bread wheat yield through modulating an unselected AP2/ERF gene. 
    <span class="xie">Nature Plants. </span> (2022).
    <br><a href="https://www.nature.com/articles/s41477-022-01197-9">doi:10.1038/s41477-022-01197-9.</a>
</p>

<p>
    12. Xiaofei Yang, Haopeng Yu, Wenqing Sun, Ling Ding, Ji Li, Jitender Cheema, Ricardo Ramirez-Gonzalez, Xuebo Zhao, Azahara C. Martín, <span class="bold">Fei Lu</span>, Bao Liu, Cristobal Uauy, Yiliang Ding*, Huakun Zhang*. 
    Wheat in vivo RNA structure landscape reveals a prevalent role of RNA structure in modulating translational subgenome expression asymmetry. 
    <span class="xie">Genome Biology. </span>(2021).
    <br><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02549-y">doi: 10.1186/s13059-021-02549-y</a>
</p>

<p>
    11. Xiao Zhang*, Yonghui Zhu, Karl A. G. Kremling, M. Cinta Romay, Robert Bukowski, Qi Sun, Shibin Gao, Edward S. Buckler*, <span class="bold">Fei Lu*.</span> 
    Genome‑wide analysis of deletions in maize population reveals abundant genetic diversity and functional impact. 
    <span class="xie">Theor. Appl. Genet. </span> (2021).
    <br><a href="https://link.springer.com/article/10.1007/s00122-021-03965-1">doi:10.1007/s00122-021-03965-1</a>
</p>

<p>
    10. Jing Wang, Jun Xu, Xiaohan Yang, Song Xu, Ming Zhang, <span class="bold">Fei Lu*</span>.
    Boosting the power of transcriptomics by developing an efficient gene expression profiling approach.
    <span class="xie"> Plant Biotechnol Jounrnal </span> (2021)
    <br><a href="https://onlinelibrary.wiley.com/doi/full/10.1111/pbi.13706">doi:10.1111/PBI.13706</a>
</p>


<p>
    9. Xuebo Zhao, Xiangdong Fu, Changbin Yin*, <span class="bold">Fei Lu*</span>.
    Wheat Speciation and Adaptation: Perspectives from Reticulate Evolution. 
    <span class="xie">aBiotech.</span> (2021)
    <br><a href="https://link.springer.com/article/10.1007/s42994-021-00047-0"> doi:10.1007/s42994-021-00047-0</a>
</p>


<p>
    8. Huijie Zhai, Congcong Jiang, Yue Zhao, Shuling Yang, Yiwen Li, Kunfang Yan, Shuyu Wu, Bingke Luo, Yi Du, Huaibing Jin, Xin Liu, Yanbin Zhang, <span class="bold">Fei Lu</span>, 
   Wenjing Hu, Jiangchun Wang, Haitao Gao, Guihong Yin, Kunpu Zhang*, Guangwei Li*, Daowen Wang*. 
    Wheat heat tolerance is impaired by heightened deletions in the distal end of 4AL chromosomal arm. 
    <span class="xie">Plant Biotechnol. J. </span> (2020).
    <br><a href="https://onlinelibrary.wiley.com/doi/10.1111/pbi.13529">doi: https://doi.org/10.1111/pbi.13529</a>
</p>

<p>
    7. Yao Zhou#, Xuebo Zhao#, Yiwen Li, Jun Xu, Aoyue Bi, Lipeng Kang, Daxing Xu, Haofeng Chen, Ying Wang, Yuan-ge Wang, Sanyang Liu, Chengzhi Jiao, Hongfeng Lu, Jing Wang, Changbin Yin, Yuling Jiao*, <span class="bold">Fei Lu*.</span>
    <span class="xie2">Triticum </span> sequencing provides insights into wheat adaptation. 
    <span class="xie"> Nature Genetics. </span> (2020). 
    <br><a href="https://www.nature.com/articles/s41588-020-00722-w">doi:10.1038/s41588-020-00722-w</a>
</p>

<p>
    6. Chunzhi Zhang, Pei Wang, Die Tang, Zhongmin Yang, <span class="bold">Fei Lu</span>, Jianjian Qi, Nilesh R. Tawari, Yi Shang, Canhui Li & Sanwen Huang. 
    The genetic basis of inbreeding depression in potato.
    <span class="xie"> Nature Genetics. </span> (2019). 
    <br><a href="https://www.nature.com/articles/s41588-018-0319-1">doi:10.1038/s41588-018-0319-1</a>
</p>

<p>
    5. Qun Lian, Die Tang, Zhiyan Bai, Jianjian Qi, <span class="bold">Fei Lu</span>, Sanwen Huang, Chunzhi Zhang*. 
    Acquisition of deleterious mutations during potato polyploidization.
    <span class="xie"> J Integr Plant Biol. </span> (2019). 
    <br><a href="https://onlinelibrary.wiley.com/doi/10.1111/jipb.12748"> https://doi.org/10.1111/jipb.12748</a>
</p>

<p>
    4. Springer Nathan M., Anderson Sarah N., Andorf Carson M., Ahern Kevin R., Bai Fang, Barad Omer, Barbazuk W. Brad, Bass Hank W., Baruch Kobi, Ben-Zvi Gil, Buckler Edward S.,
     Bukowski Robert, Campbell Michael S., Cannon Ethalinda K. S., Chomet Paul, Dawe R. Kelly, Davenport Ruth, Dooner Hugo K., Du Limei He, Du Chunguang, Easterling Katherine A.,
      Gault Christine, Guan Jiahn-Chou, Hunter Charles T., Jander Georg, Jiao Yinping, Koch Karen E., Kol Guy, Köllner Tobias G., Kudo Toru, Li Qing, <span class="bold">Lu Fei</span>, 
      Mayfield-Jones Dustin, Mei Wenbin, McCarty Donald R., Noshay Jaclyn M., Portwood John L., Ronen Gil, Settles A. Mark, Shem-Tov Doron, Shi Jinghua, Soifer, Ilya, Stein Joshua C., 
      Stitzer Michelle C., Suzuki Masaharu, Vera Daniel L., Vollbrecht Erik, Vrebalov Julia T., Ware Doreen, Wei Sharon, Wimalanathan, Kokulapalan, Woodhouse Margaret R., Xiong Wenwei, Brutnell Thomas P. 
      The maize W22 genome provides a foundation for functional genomics and transposon biology. 
      <span class="xie"> Nature Genetics. </span> (2018).
      <br><a href="https://www.nature.com/articles/s41588-018-0158-0 ">doi:10.1038/s41588-018-0158-0 </a>
</p>

<p>
    3. Sun Silong, Zhou Yingsi, Chen Jian, Shi Junpeng, Zhao Haiming, Zhao Hainan, Song Weibin, Zhang Mei, Cui Yang, Dong Xiaomei, Liu Han, 
    Ma Xuxu, Jiao Yinping, Wang Bo, Wei Xuehong, Stein Joshua C., Glaubitz Jeff C., <span class="bold">Lu Fei</span>, Yu Guoliang, 
    Liang Chengzhi, Fengler Kevin, Li Bailin, Rafalski Antoni, Schnable Patrick S., Ware Doreen H., Buckler Edward S., Lai Jinsheng. 
    Extensive intraspecific gene order and gene structural variations between Mo17 and other maize genomes.
    <span class="xie"> Nature Genetics. </span> (2018).
    <br><a href="https://doi.org/10.1038/s41588-018-0182-0">doi:10.1038/s41588-018-0182</a>
</p>

<p>
    2. Karl A. G. Kremling, Shu-Yun Chen, Mei-Hsiu Su, Nicholas K. Lepak, M. Cinta Romay, Kelly L. Swarts, <span class="bold">Fei Lu</span>, Anne Lorant, Peter J. Bradbury & Edward S. Buckler. 
    Dysregulation of expression correlates with rare-allele burden and fitness loss in maize. 
    <span class="xie"> Nature. </span> (2018).
    <br><a href="https://www.nature.com/articles/nature25966">doi:10.1038/nature25966</a>
</p>

<p>
    1. Punna Ramu*, Williams Esuma, Robert Kawuki, Ismail Y Rabbi, Chiedozie Egesi, Jessen V Bredeson, Rebecca S Bart, Janu Verma, Edward S Buckler, <span class="bold">Fei Lu*</span>. 
    Cassava haplotype map highlights fixation of deleterious mutations during clonal propagation. 
    <span class="xie"> Nature Genetics. </span> (2017).
    <br><a href="https://www.nature.com/articles/ng.3845">doi:10.1038/ng.3845</a>
</p>
<br>
<br>
<br>
<br>
